src/golang/config/config.go

```diff
@@ -29,5 +32,17 @@ func ParseServerConfiguration(confPath string) *ServerConfiguration {
 		os.Exit(1)
 	}
 
+	if config.StorageConfig == nil {
```

Comment:
Is this something necessary to make s3 working or it's a nice fix we add on the way?

Reply:
So it's necessary because we are adding a new field to the `config.yml` file. For users that don't run `aqueduct storage` that field won't exist, so we'd get a null pointer exception here without the check.